src/extensions/default/CodeFolding/main.js

```diff
@@ -317,29 +416,12 @@ define(function (require, exports, module) {
             init();
```

Comment:
Btw, why do we need to call `init()` again on project open?  Now that we're listening for preference changes at all times, it seems like this should be unnecessary?